x-pack/plugins/security_solution/public/common/containers/source/index.tsx

```diff
@@ -75,7 +62,7 @@ export const getIndexFields = memoizeOne(
 );
 
 export const getBrowserFields = memoizeOne(
-  (_title: string, fields: IndexField[]): BrowserFields =>
```

Comment:
I think _title arguments was there to make the memoization simpler and faster than an object with a different reference